src/esp/bindings/SimBindings.cpp

```diff
@@ -379,6 +379,9 @@ void initSimBindings(py::module& m) {
           R"(List of sensor specifications for one simulator. For batch rendering, all simulators must have the same specification.)")
       .def_readwrite("gpu_device_id", &ReplayRendererConfiguration::gpuDeviceId,
                      R"(The system GPU device to use for rendering)")
+      .def_readwrite("enable_frustum_culling",
```

Comment:
Perhaps we can rename this to something like 'force_culling_off' or 'force_disable_culling' so there's no ambiguity in its purpose.

Reply:
My two cents: a flag named "enable..." is more intuitive than "disable...". I prefer Mikael's name here. Of course it should default to true for this feature.

Reply:
The purpose is to force it off if it isn't working, though, right?

Reply:
The `SimulatorConfiguration` semantic is the same than the one suggested here.

https://github.com/facebookresearch/habitat-sim/blob/main/src/esp/sim/SimulatorConfiguration.h#L36
```
//! Enable or disable the frustum culling optimisation
bool frustumCulling = true;
```